src/main/kotlin/biokotlin/util/MergeGVCFUtils.kt

```diff
@@ -167,20 +167,46 @@ private fun createSNP(
 
                     when {
 
-                        variant.isINS -> {
-                            symbolicAlleles.add("<INS>")
-                            Pair(variant.isPhased(0), listOf("<INS>"))
-                        }
+                        variant.isIndel -> {
```

Comment:
Do you have a unit test to verify that this change works correctly?

Reply:
Working on it. Also, I might have committed these changes to master. Can you tell if I did that?

Reply:
Yeah it looks like there was 3 commits to master.  It's odd though that you still have a diff here then if it matches.

Reply:
Do you have suggestion how to proceed?